demo/src/Types.elm

```diff
@@ -54,6 +56,33 @@ to the program as `endpoint` (see above).
 type Msg
     = Endpoint
     | GotQuotes (Result Http.Error (List Quote))
+    | RandomNumber Int
+
+
+{-| Your custom interop type.
+
+Should enumerate the interop (JavaScript) functions which can be called.
+
+-}
+type Interop
+    = GetRandom Int
+
```

Comment:
This is an enumeration of the kinds of interop functions we can call, with signatures